packages/shared/src/db/schema.ts

```diff
@@ -226,6 +230,137 @@ export const chatMembers = sqliteTable(
   }
 );
 
+export const groupFlaggedContent = sqliteTable(
+  'group_flagged_content',
+  {
+    groupId: text('group_id')
+      .references(() => groups.id)
+      .notNull(),
+    postId: text('post_id').notNull(),
+    channelId: text('channel_id').notNull(),
+    flaggedByContactId: text('flagged_by_contact_id').notNull(),
+    flaggedAt: timestamp('flagged_at'),
+  },
+  (table) => {
+    return {
+      pk: primaryKey({
+        columns: [table.groupId, table.postId],
+      }),
+    };
+  }
+);
+
+export const groupFlaggedContentRelations = relations(
+  groupFlaggedContent,
+  ({ one }) => ({
+    group: one(groups, {
+      fields: [groupFlaggedContent.groupId],
+      references: [groups.id],
+    }),
+    post: one(posts, {
+      fields: [groupFlaggedContent.postId],
+      references: [posts.id],
+    }),
+    channel: one(channels, {
+      fields: [groupFlaggedContent.channelId],
+      references: [channels.id],
+    }),
+    flaggedBy: one(contacts, {
+      fields: [groupFlaggedContent.flaggedByContactId],
+      references: [contacts.id],
+    }),
+  })
+);
+
+export const groupMemberInvites = sqliteTable(
+  'group_member_invites',
+  {
+    groupId: text('group_id')
```

Comment:
Is this compatible with @latter-bolden's stuff?

Reply:
Good question, will have to look.

Reply:
With a few changes I've made, this should be fine